sensor_msgs/test/test_pointcloud_conversion.cpp

```diff
@@ -18,7 +18,26 @@
 #include "sensor_msgs/msg/point_cloud.hpp"
 #include "sensor_msgs/msg/point_cloud2.hpp"
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#else
+#pragma GCC diagnostic push
+#ifndef __clang__
+#pragma GCC diagnostic ignored "-Wcpp"
+#endif
+#define RCUTILS_SKIP_WARNING
```

Comment:
is this actually a macro defined in `rcutils`?

If not, I would use a name more specific to this package.

Reply:
Yeah, I was originally going to try to prototype it here and then move it upstream, but it go so complicated I'm going to need to defer the upstream proposal until after the freeze. I'll rename it for clarity for now. And we can replace it with a potential macro.